### PR TITLE
Add guag.es integration.

### PR DESCRIPTION
**Pull request by markcaudill**
Set `GUAGES` in `pelicanconf.py` to your `data-site-id`.

See http://get.gaug.es/

### CODE REVIEW

**Comment by alexandrevicenzi** [comment]
Hi @markcaudill, Thanks.

Just a small change, could you place the `if` in the `include`?  This will avoid loading a template that doesn't use anything.

File: templates/base.html [hunk 1]
```diff
@@ -94,6 +94,7 @@
     </footer>
   </main>
   {% include "partial/ga.html" %}
+  {% include "partial/guages.html" %}
```
Comment (alexandrevicenzi) [inline templates/base.html]: place if here

File: templates/partial/guages.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{% if GUAGES %}
```
Comment (alexandrevicenzi) [inline templates/partial/guages.html]: remove from here

**Comment by markcaudill** [comment]
Ok, how about that? I was initially attempting to follow the precedent set by the Google Analytics integration. But it looks like the Piwik integration is doing its own thing too! Anyway, great theme. Thanks for adding this.

**Comment by alexandrevicenzi** [comment]
Thanks for this new integration @markcaudill.
The others are a little bit messy. I'll change they to.

**Comment by alexandrevicenzi** [comment]
@markcaudill, could you update the [wiki](https://github.com/alexandrevicenzi/Flex/wiki)?

**Pull request merged**